lib/dyn.ml

```diff
@@ -55,6 +55,8 @@ sig
   include PreS
 
   module Easy : sig
+
+    val make_dyn_tag : string -> ('a -> t) * (t -> 'a) * 'a tag
```

Comment:
I don't think it's worthwhile to export a function `make_dyn_tag` when it seems it's only used once and you can reimplement it using the basic blocks provided out of the `Easy` module. This module is there mostly for backward compatibility, and the fact we expose a tag in the signature of one of its functions while there is no other "easy" function manipulating tags makes this even more obvious.

Reply:
I think the function is fine as indeed otherwise `Easy` is not a self-contained module. Additionally, it really lowers the cost of porting code to be typed.